lib/config.js

```diff
@@ -38,6 +38,11 @@ var Config = module.exports = function (cfg) {
   this.scripts = this._loadScripts(this.archetypes);
 };
 
+// Expose `require()` for testing.
```

Comment:
I understand why this is being done, I was just suggesting a tiny bit more explanation in the comment in *this* file.